winui/Rating/ToolTip.md

```diff
@@ -1,12 +1,92 @@
 ---
 layout: post
 title: ToolTip with WinUI Rating control | Syncfusion
-description: Learn here all about Enabletooltip and tooltip format support in the rating items in WinUI rating control.
+description: Learn here all about Enable tooltip features and tooltip format support in the rating items in WinUI rating control.
 platform: WinUI
 control: SfRating
 documentation: ug
 ---
 
 # ToolTip in WinUI Rating
 
-This section explains the tooltip features available in the WinUI Rating control.
+The Rating control EnableToolTip can be customized using the `EnableToolTip` property. The default value is `false`.
+
+{% tabs %}
```

Comment:
The Rating control tooltip can be enabled by using the `EnableToolTip` property. It will show the rating value content as a tooltip when the mouse hovering on the control. The default value is `false`.

Reply:
Changed the content as mentioned above.